Android/GoogleBillingClient/source/GoogleBillingClient/GoogleBillingClient.csproj

```diff
@@ -35,15 +35,15 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="..\..\externals\billing.aar">
-      <Link>billing.jar</Link>
+    <None Include="..\..\externals\billing-3.0.3.aar">
+      <Link>billing-3.0.3.aar</Link>
```

Comment:
Doe this file need to be listed in the project twice? The one `@(LibraryProjectZip)` seems like it would enough.